RunAnalysis/DatasetsPaths.py

```diff
@@ -15,7 +15,7 @@ def findMainPath():
 
 # Set up the example path for the analysis depending on the user machine name.
 # If the user is not in the list, add the default path.
-username = os.environ['USER']
+username = "user"
```

Comment:
This thing does not work for you? What do you get from `os.environ['USER']`?

Reply:
This works when using the interactive condor environment, but fails when queueing a job using condor_submit

Reply:
And what happens there? Does it give you a weird result or it fails?

Reply:
OK, I have fixed it - previously I was starting the env by running the conda activate command in a bash script, which i guess doesn't add the user, but adding getenv=True to the condor submit file to start the env does. I'll revert the change